### ISSUE TITLE
[FEATURE] Probability Calibration

### ISSUE THREAD

**Issue by zimratK** [scikit-multiflow/scikit-multiflow#287]
**Is your feature request related to a problem? Please describe.**
I've trained several classifiers using different methods from the library, and I tried to get the probabilities of my test set predictions   
using *predict_proba*.  The results I got were not "meaningful", I got discrete values (such as 0, 0.2, 0.4, 0.6, 0.8, 1) or ones and zeros only.

**Describe the solution you'd like**
In sklearn there is a class called *CalibratedClassifierCV* for probability calibration with isotonic regression or logistic regression.
A similar solution in scikit-multiflow may be very helpful.
